files/class-api-cache.php

```diff
@@ -95,5 +120,10 @@ public function remove_file( $filepath ) {
 			unlink( $this->files[ $file_name ] );
 			unset( $this->files[ $file_name ] );
 		}
+
+		// Remove cached stats too if any
+		if ( isset( $this->file_stats[ $file_name ] ) ) {
+			unset( $this->file_stats[ $file_name ] );
```

Comment:
Should this be encapsulated into a reusable function, like we have for `cache_file_stats` and `get_file_stats`?

Reply:
I'm not sure if that's necessary. It will basically be a one line function wouldn't it?